dtfs-central-api/src/v1/controllers/utilisation-report-service/get-utilisation-report-reconciliation-details-by-id.controller/helpers/map-to-fee-record-utilisation.ts

```diff
@@ -0,0 +1,50 @@
+import { FeeRecordEntity, TfmFacility, FeeRecordUtilisation } from '@ukef/dtfs2-common';
+import Big from 'big.js';
+import { getLatestCompletedAmendmentToFacilityValue } from '../../../../../helpers';
+
+/**
+ * Calculates the exposure based on the current utilisation
+ * @param utilisation - The current utilisation of the facility
+ * @param coverPercentage - The cover percentage of the facility
+ * @returns The exposure
+ */
+export const calculateExposure = (utilisation: number, coverPercentage: number) => {
+  return new Big(utilisation).mul(coverPercentage).div(100).round(2).toNumber();
+};
+
+/**
+ * Maps a fee record entity and the tfm facility it corresponds to to the
```

Comment:
typo.  double to to

Reply:
Not a typo, will add a comma and moved the line break to make it more readable